JuiceMaker/JuiceMaker/Controller/ViewController.swift

```diff
@@ -1,18 +1,80 @@
 //
 //  JuiceMaker - ViewController.swift
-//  Created by yagom. 
+//  Created by uemu, Toy.
 //  Copyright © yagom academy. All rights reserved.
 // 
 
 import UIKit
 
-class ViewController: UIViewController {
+final class ViewController: UIViewController {
+    
+    @IBOutlet weak var strawberryLabel: UILabel!
+    @IBOutlet weak var bananaLabel: UILabel!
+    @IBOutlet weak var pineappleLabel: UILabel!
+    @IBOutlet weak var mangoLabel: UILabel!
+    @IBOutlet weak var kiwiLabel: UILabel!
+    
+    var juiceMaker = JuiceMaker()
+    var fruitStore = FruitStore.shared
+    var fruitJuice: String?
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        showStock()
     }
 
-
+    func showStock() {
+        fruitStore.fruitQuantity.forEach{(key, value) in
+            switch key {
+            case .strawberry:
+                strawberryLabel.text = String(value)
+            case .banana:
+                bananaLabel.text = String(value)
+            case .kiwi:
+                kiwiLabel.text = String(value)
+            case .mango:
+                mangoLabel.text = String(value)
+            case .pineapple:
+                pineappleLabel.text = String(value)
+            }
+        }
+    }
+    
+    @IBAction func manufactureJuice(_ sender: UIButton) {
+        guard let order = sender.currentTitle else { return }
+        guard let juice = FruitJuice.Order(rawValue: order)?.juiceOrder else { return }
```

Comment:
```suggestion
        guard let order = sender.currentTitle,
              let juice = FruitJuice.Order(rawValue: order)?.juiceOrder else { return }
```

이렇게 한 번에 바인딩을 진행할 수 도 있습니다.